types/consola.d.ts

```diff
@@ -125,9 +125,15 @@ export declare class Consola {
   mock(mockFn: ConsolaMockFn): any
 }
 
+interface BasicReporterFormatOptions {
+    date?: boolean,
+    colors?: boolean,
+    compact?: boolean
+}
```

Comment:
just a tiny style suggestion
```suggestion
interface BasicReporterFormatOptions {
  date?: boolean,
  colors?: boolean,
  compact?: boolean
}
```